tests/test_static_files.py

```diff
@@ -8,16 +8,8 @@
 
 
 def setup_app(app):
-    @app.route('/')
```

Comment:
So there is no problem here like you suspected in #136 ?

Reply:
I don't know why I saw failed tests. I can't reproduce it now. So I didn't create an issue and just fixed test. Commit has short description what happened.

Reply:
Could you add something like this comment to the commit message? "By mistake" is not much information in case someone is wondering about this commit in a year

Reply:
Fixed